### DISCUSSION TITLE
Field `image` adds `/storage//` before image url.

### DISCUSSION THREAD

**Discussion by hagealex** [Laravel-Backpack/community-forum#1128] [Q&A (Help)] [answered]
Hello!

I make use of the `image` field type: 

```
 [
    'name'  => 'image_url',
    'label' => 'Bild/Logo für die Serie',
    'type'  => 'image',
 ],
```

I store the images in Cloudinary. I have this attribute setter on my model:

```
public function setImageUrlAttribute($value): void
    {
        $this->handleImageChange($value);
    }
 ```

This handles the upload to Cloudinary, takes the image url from cloudinary and stores this url for the `image_url` field. In my Database the value is correct.

But when, after the succesful upload, I go back to the edit view, the image src looks like this: `/storage//https://res.cloudinary.com/here-comes-the-image-url`

So before the url there is a `/storage//` added. Therefor the image is not displayed. When I remove this using the browser console the image shows up.

Does someone have an idea what I can do to prevent Backpack adding the `/storage//` prefix?

Kind regrads

**Comment by karandatwani92** [comment, marked as answer]
Hey @hagealex 

The problem is that you have not configured a disk for `cloudinary`.  Disks can be added at `config/filesystems.php`.
A `cloudinary` driver will be required to add a `cloudinary` disk. This is how the [Laravel File Storage](https://laravel.com/docs/11.x/filesystem) system works.

I found two packages to help you with `cloudinary ` Laravel integration that comes with `cloudinary` filesystem/driver.
- https://github.com/cloudinary-community/cloudinary-laravel
- https://github.com/yoelpc4/laravel-cloudinary

Install the one you like, and configure the API keys. Then you can use it like this:
```
 [
    'name'  => 'image_url',
    'label' => 'Bild/Logo für die Serie',
    'type'  => 'image',
    'disk' => 'cloudinary'
 ],
```

You can also make `cloudinary` a default Filesystem Disk for your Laravel app via `config/filesystems.php` config.

**Reply by hagealex** [reply]
@karandatwani92 This solved my issue! Didn't know about the filesystem configurations before. Thank you so much!

**Reply by karandatwani92** [reply]
Oh! Nice to know this... If you like Backpack so far, please give it a star on the [Github repo](https://github.com/Laravel-Backpack/CRUD). Thanks
